### PR TITLE
Add 17.06.1 release notes

### PR DESCRIPTION
**Pull request by mstanleyjones**
Add release for 17.06.1 and refresh the CLI for stable and edge directories

cc/ @andrewhsu

### CODE REVIEW

**Comment by mstanleyjones** [comment]
@thaJeztah I'll look at what needs to change to vendor the `dockerd` reference instead of using the fake YAML. cc/ @johndmulhausen

File: _data/engine-cli-edge/docker_build.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ long: |-
   repository acts as the build context. The system recursively fetches the
   repository and its submodules. The commit history is not preserved. A
   repository is first pulled into a temporary directory on your local host. After
-  that succeeds, the directory is sent to the Docker daemon as the context.
+  the that succeeds, the directory is sent to the Docker daemon as the context.
```
Comment (thaJeztah) [inline _data/engine-cli-edge/docker_build.yaml]: 🤔 looks like I need to fix this upstream 😄

File: _data/engine-cli-edge/docker_exec.yaml [hunk 2]
```diff
@@ -41,53 +41,23 @@ options:
...
-  1ae3b36715d2        ubuntu:latest       "bash"              17 seconds ago      Up 16 seconds (Paused)                       test
-
-  $ docker exec test ls
-
-  FATA[0000] Error response from daemon: Container test is paused, unpause the container before exec
-
-  $ echo $?
-  1
-  ```
+examples: "### Run `docker exec` on a running container\n\nFirst, start a container.\n\n```bash\n$
```
Comment (thaJeztah) [inline _data/engine-cli-edge/docker_exec.yaml]: Interesting; do we know why this changed from the `|-` format to plain `""` ?

**Comment by mstanleyjones** [comment]
@thaJeztah the swagger files are fetched at build time. It's still fetched from the `17.06` branch, so no change at this time. Unless there is yet another new API version for this?

File: _data/engine-cli-edge/docker_exec.yaml (hunk 2 above)
Comment (mstanleyjones) [inline _data/engine-cli-edge/docker_exec.yaml]: No idea.

**Comment by mstanleyjones** [comment]
Just pushed a commit that pulls the dockerd.md files from upstream.

**Comment by mstanleyjones** [comment]
@thaJeztah let me know when the fixes have been done upstream and I will amend the CLI commit to have the improved output.

**Comment by thaJeztah** [comment]
@mstanleyjones feel free to merge before that, I'd have to find a spot to make the change (and change is only a minor typo)

**Pull request merged**